### PR TITLE
feat: Smart Sessions - Session List

### PR DESCRIPTION
**Pull request by tomiir**
# Description
- Adds a SmartSession list screen where a user can visualize all Active,Revoked and Expired Smart Sessions.

## Type of change

- [ ] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Associated Issues

Closes APKT-1339
Closes APKT-1340

# Checklist

- [X] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [X] My changes generate no new warnings
- [X] I have reviewed my own code
- [X] I have filled out all required sections
- [ ] I have tested my changes on the preview link
- [ ] Approver of this PR confirms that the changes are tested on the preview link

### CODE REVIEW

**Review by magiziz — APPROVED** [review]

**Review by lukaisailovic — APPROVED** [review]

File: packages/experimental/src/smart-session/ui/views/w3m-smart-session-list-view/index.ts [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+import styles from './styles.js'
+import { SmartSessionsController } from '../../../controllers/SmartSessionsController.js'
+import { DateUtil } from '@reown/appkit-common'
+import type { SmartSession } from '../../../utils/TypeUtils.js'
+
+const TABS = 3
+const TABS_PADDING = 48
+const MODAL_MOBILE_VIEW_PX = 430
+const SMART_SESSION_TABS = [{ label: 'Active' }, { label: 'Expired' }, { label: 'Revoked' }]
+@customElement('w3m-smart-session-list-view')
```
Comment (enesozturk) [inline packages/experimental/src/smart-session/ui/views/w3m-smart-session-list-view/index.ts:14]: Could separate the blocks with empty line
```suggestion

@customElement('w3m-smart-session-list-view')
```

File: packages/experimental/src/smart-session/utils/TypeUtils.ts [hunk 2]
```diff
@@ -132,3 +132,26 @@ export type ActivatePermissionsRequest = {
   pci: string
   context: `0x${string}`
 } & AddPermissionRequest
+
+// T0DO: Fetch right format from max
```
Comment (enesozturk) [inline packages/experimental/src/smart-session/utils/TypeUtils.ts:136]: Lil hack 🤡

Comment (enesozturk) [reply packages/experimental/src/smart-session/utils/TypeUtils.ts:136]: Should these be done before merging this?

File: packages/scaffold-ui/src/partials/w3m-header/index.ts [hunk 3]
```diff
@@ -167,6 +168,21 @@ export class W3mHeader extends LitElement {
...
+    }
+
+    return html`<wui-flex>
+      <wui-icon-link
+        icon="clock"
+        @click=${() => RouterController.push('SmartSessionList')}
+        data-testid="w3m-header-smart-sessions"
+      ></wui-icon-link>
+      ${this.closeButtonTemplate()}
+    </wui-flex> `
```
Comment (tomiir) [inline packages/scaffold-ui/src/partials/w3m-header/index.ts:183]: This is wrong, `SmartSessionList` will not be available unless the dapp imports the package. 
I should hide this with a feature flag.

Comment (tomiir) [reply packages/scaffold-ui/src/partials/w3m-header/index.ts:185]: Done

**Pull request merged**